sbol2/property.py

```diff
@@ -761,6 +761,8 @@ def setSinglePropertyValue(self, new_value):
         # TODO: This can leave the attribute empty if `add` fails.
         # Can we capture that and sent the old value back again?
         self._sbol_owner.owned_objects[self._rdf_type].clear()
+        if new_value is None:
+            return
```

Comment:
This will bypass the validation rules, which are fired at the end of the `add` method. Is that what we want? I feel like we want setting to `None` to still fire any associated validation rules. Do you agree? Or is that not necessary.

Reply:
Yeah the fix above is too simplistic.  In principle, it's probably a good idea to run validation rules when we unset or remove a value.  This probably boils down to seeing what happens with `self.validate(None)`. That might break some stuff.

Also instead of handling the unsetting by the `add` method, however, it would make more sense to handle the unsetting with the `remove` method.  But currently the `remove` methods don't currently invoke the validation rules.

Reply:
OK, 5ef42bb takes a different approach to unsetting